packages/mobile/branding/celo/src/brandingConfig.ts

```diff
@@ -12,5 +12,5 @@ export const FUNDING_LINK = 'https://celo.org/fund-wallet'
 export const CELO_REWARDS_LINK = 'https://celo.org/support'
 export const FORUM_LINK = 'https://forum.celo.org/c/support'
 export const CELO_SUPPORT_EMAIL_ADDRESS = 'support@celo.org'
-export const SUPERCHARGE_T_AND_C = 'https://valoraapp.com/supercharge-terms-and-conditions'
+export const SUPERCHARGE_LEARN_MORE = 'https://valoraapp.com/supercharge-terms-and-conditions'
```

Comment:
should this link be `https://valoraapp.com/support/supercharge-learn-more`? it will not be replaced by the link from the branding repo for external contributors otherwise?

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2348/commits/de378fe6ce61021d5703c1b23aea7d420d6cd6e4!